src/transfer-manager.ts

```diff
@@ -396,12 +396,14 @@ export class TransferManager {
         ...options.passthroughOptions,
       };
 
-      passThroughOptionsCopy.destination = filePath;
+      passThroughOptionsCopy.destination = filePath
+        .split(path.win32.sep)
```

Comment:
Perhaps using the OS-specific `path.sep` would be preferred to prefer POSIX customers' ability to write with `\`.

```suggestion
        .split(path.sep)
```